### DISCUSSION TITLE
Add additional share actions

### DISCUSSION THREAD

**Discussion by iamjonny** [community/community#19849] [Mobile]
Currently sharing content (eg via share option on a tab in edge browser) allows you to create a new issue

Just learning more about the (now!) Obvious differences between discussions and issues, and would be great to

- [ ] share as new Discussion

This could be an additional optional registered share Github action, although that could be confusing unless the icon lets user identify  **issue** option, Vs new discussion option

Alternatively the ui could allow the user to select type of newly created object when sharing, via the existing single git share option

Also would be great to have both as app context shortcuts (the options you get when long pressing the app shortcut, like Spotify and YouTube make use of)

**Comment by elisealix22** [comment]
@iamjonny thank you again for another amazing feature request! I just opened a tracking issue with our Android team specifically. Please keep the feedback coming!
